MAVProxy/modules/mavproxy_steamvr.py

```diff
@@ -0,0 +1,304 @@
+"""
+use SteamVR tracker data to provide VISION_POSITION_ESTIMATE and GPS_INPUT data
+"""
+
+import math
+import threading
+import time
+
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+from MAVProxy.modules.lib import LowPassFilter2p
```

Comment:
I can't find this module in MAVProxy. Did you mean to add this module too?

Reply:
Opps, that is left over from vicom, i used that module as a starting point